File: src/main/java/bg/fmi/rateuni/dto/response/UserDetail.java
package bg.fmi.rateuni.dto.response;

import lombok.Data;

import java.util.List;
import java.util.UUID;

@Data
public class UserDetail {
    private UUID id;
    private String email;
    private String facultyNumber;
    private String universityName;
    private String facultyName;
    private String programmeName;
    private List<DisciplineResponse> disciplines;
    private List<ReviewResponse> reviews;
}

File: front-end/src/app/dto/response/user-detail.ts
import {DisciplineResponse} from "./discipline-response";
import {ReviewResponse} from "./review-response";

export class UserDetail {
    id: string;
    email: string;
    facultyNumber: string;
    universityName: string;
    facultyName: string;
    programmeName: string;
    disciplines: DisciplineResponse[];
    reviews: ReviewResponse[];
}
